gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/tpgm/aggregation/MinMaxTimeTest.java

```diff
@@ -159,12 +162,22 @@ public void testAggregationFunctionsWithAllDefaults() throws Exception {
       new MinTime(keyMin, temporalAttribute, field),
       new MaxTime(keyMax, temporalAttribute, field));
     TemporalGraphHead head = result.getGraphHead().collect().get(0);
-    // The expected values for max and min aggregations. Those should be null, when the minimum
-    // of all FROM or the maximum of all TO fields is calculated.
-    final PropertyValue min = field == FROM ? PropertyValue.NULL_VALUE :
-      PropertyValue.create(MAX_VALUE);
-    final PropertyValue max = field == TO ? PropertyValue.NULL_VALUE :
-      PropertyValue.create(MIN_VALUE);
+
+    // The expected values for max and min aggregations. For valid times, they should be null.
+    PropertyValue min = PropertyValue.NULL_VALUE;
+    PropertyValue max = PropertyValue.NULL_VALUE;
```

Comment:
I think you can replace those variables with just one, since they are always set to the same value.